test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -18,7 +19,7 @@ public NormalizerTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact]
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]
```

Comment:
What about this test and the below test require x64? I don't see tensorflow or lightgbm in the tests. Are there other things that only work on x64?

Reply:
As mentioned here: https://github.com/dotnet/machinelearning/pull/1008#issuecomment-424145620, there are a few tests which have different outputs from the baseline.

These need further investigation and potentially fixes.